osu.Game/Screens/Play/SaveFailedScoreButton.cs

```diff
@@ -87,5 +96,30 @@ private void load(OsuGame? game, Player? player, RealmAccess realm)
                 }
             }, true);
         }
+
+        public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
+        {
+            switch (e.Action)
+            {
+                case GlobalAction.SaveReplay:
+                    button.TriggerClick();
+                    return true;
+
+                case GlobalAction.ExportReplay:
+                    Task.Run(importFailedScore).ContinueWith(t =>
+                    {
+                        importedScore = realm.Run(r => r.Find<ScoreInfo>(t.GetResultSafely().ID)?.Detach());
+                        Schedule(() => state.Value = importedScore != null ? DownloadState.LocallyAvailable : DownloadState.NotDownloaded);
+                        scoreManager.Export(importedScore);
+                    });
```

Comment:
Will raise unobserved exceptions if anything goes wrong here. Should probably be concluded with a `.FireAndForget()`.

Reply:
This was already in `master`, but yeah seems like we should have `FireAndForget`